Classes/Team_GameBase.uc

```diff
@@ -1247,13 +1247,13 @@ function int ReduceDamageOld(int Damage, pawn injured, pawn instigatedBy, vector
             }
             else if(DamageType == class'DamTypeLinkPlasma')
             {
-                PRI.Link.Secondary.Hit++;
-                PRI.Link.Secondary.Damage += Damage;
+                PRI.Link.Primary.Hit++;
+                PRI.Link.Primary.Damage += Damage;

```

Comment:
Swap hit and damage counts for TAM. Other file is the AM mod.

Reply:
We would also want to keep the AM Mod updated to match TAM for any changes we make.  Previous work has neglected the mod and took a while to catch it back up to where it is now.

Im about to test this changes tomorrow morning on a test build on my end.  Will post with results / push this through if it works as desired!